ksqldb-engine/src/test/java/io/confluent/ksql/security/KsqlAuthorizationValidatorFactoryTest.java

```diff
@@ -176,6 +178,22 @@ public void shouldReturnEmptyValidatorIfAuthorizedOperationsReturnNull() {
     assertThat(validator, is(Optional.empty()));
   }
 
+  @Test
+  public void shouldReturnEmptyValidatorIfKafkaBrokerVersionTooLow() {
```

Comment:
this test is currently broken, I didn't do proper setup - I'm looking into it now